app/api/entities/specs/entities.spec.js

```diff
@@ -535,6 +535,18 @@ describe('entities', () => {
           .catch(catchErrors(done));
       });
     });
+
+    describe('save from public form', () => {
+      it('should save the entity with unrestricted access', async () => {
+        const user = {};
+        userFactory.mock(undefined);
+        const entity = { title: 'Batman begins', template: templateId, language: 'es' };
+        const createdEntity = await entities.save(entity, { user, language: 'es' });
+        expect(createdEntity._id).not.toBeUndefined();
+        expect(createdEntity.title).toEqual(entity.title);
+        userFactory.mockEditorUser();
+      });
+    });
```

Comment:
@mfacar what makes this test a  "save from public form" ? is it the fact that there is not user logged in ?

Reply:
yes @daneryl, I'm not sure if there is another place where the save method is called with an empty user, but I think there is no other way to distinguish if it comes from a public form. Maybe the name should be more descriptive in order to avoid any confusion?